File: 2543 - UFPR Gaming.c
#include <stdio.h>
int main()
{
    int i,id,n;
    int temp_id, flag, count;

    while(scanf("%d", &n) >= 0){
        scanf("%d", &id);
        count = 0;

        for(i=0;i<n;i++)
        {
            scanf("%d %d", &temp_id,&flag);
            if(flag == 0)
            {
                if(temp_id==id)
                    count++;
            }
        }

        printf("%d\n",count);



    }

    return 0;
}

File: 2543 - UFPR Gaming.py
while True:


    try:
        N_id = input()
        temp = N_id.split()
        n = int(temp[0])
        id = int(temp[1])

        count = 0

        for x in range(n):
            id_flag = input()
            temp = id_flag.split()
            flag = int(temp[1])

            if(flag == 0):
                ids = int(temp[0])
                if(ids == id):
                    count = count+1
        print(count)



    except EOFError:
        break
